bin/download_images.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+  "fmt"
+  "github.com/jessevdk/go-flags"
+  "io"
+  "io/ioutil"
+  "net/http"
+  "net/url"
+  "os"
+  "path/filepath"
+  "regexp"
+  "strings"
+)
+
+type Opts struct {
+  Directory string `short:"d" long:"directory" description:"The absolute path of the dir to scan" required:"true"`
+  ImageDir  string `short:"i" long:"image-dir" description:"The absolute path of where to put images" required:"true"`
+}
+
+func main() {
+  var opts Opts
+
+  _, err := flags.Parse(&opts)
+  if err != nil {
+    os.Exit(1)
+  }
+
+  fmt.Println("Looking for images to download")
+
+  _ = filepath.Walk(opts.Directory, func(filePath string, info os.FileInfo, err error) error {
```

Comment:
how come you inlined this function instead of defining a new function to pass to `filepath.Walk`?